src/app/components/Transaction/InfoBox.tsx

```diff
@@ -29,6 +44,14 @@ export function InfoBox({ copyToClipboard, icon: IconComponent, label, trimValue
         <Text weight="bold">{label}</Text>
         <Text>{trimValue ? trimLongString(value) : value}</Text>
       </Box>
+      {notificationVisible && (
+        <Notification
+          toast
+          status={'normal'}
+          title={t('account.addressCopied', 'Address copied.')}
```

Comment:
This isn't always an address. Maybe `{{label}} copied.`

Reply:
Indeed. Fixed.